### DISCUSSION TITLE
[Feature] Click on an image in a search result to locate its position in the album

### DISCUSSION THREAD

**Discussion by ThreeAurora** [immich-app/immich#9721] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I take a lot of pictures, but when I search for a desired result, I can't tell what else was taken that day.

The only way to find out is to copy the image path and go through it manually, which is complicated.

I wish there was a way to locate the image in the album when clicking on it in the search results.

Thanks.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
If you scroll down in the asset viewer detail panel, it already includes a list of albums that the asset is in. When clicking one, it takes you to the album. I think that covers the request right?
